From 31854ba31f6417e1d1747587a20db1124e6be5e9 Mon Sep 17 00:00:00 2001
From: rsp4jack <creepercdn@outlook.com>
Date: Wed, 14 Feb 2024 11:03:09 +0800
Subject: [PATCH] Suppress HTML header and footer for CLI highlight

---
 cli/src/main.rs | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/cli/src/main.rs b/cli/src/main.rs
index e9308628..c0f89aca 100644
--- a/cli/src/main.rs
+++ b/cli/src/main.rs
@@ -641,10 +641,6 @@ fn run() -> Result<()> {
                 highlight_options.paths,
             )?;
 
-            if html_mode && !quiet {
-                println!("{}", highlight::HTML_HEADER);
-            }
-
             let cancellation_flag = util::cancel_on_signal();
 
             let mut language = None;
@@ -733,10 +729,6 @@ fn run() -> Result<()> {
                     eprintln!("No syntax highlighting config found for path {path:?}");
                 }
             }
-
-            if html_mode && !quiet {
-                println!("{}", highlight::HTML_FOOTER);
-            }
         }
 
         Commands::Tags(tags_options) => {
-- 
2.41.0.windows.1

