From 1365a9a24d9fefb2acc1eb653d325bf820fc7bc9 Mon Sep 17 00:00:00 2001
From: rsp4jack <creepercdn@outlook.com>
Date: Wed, 14 Feb 2024 16:19:06 +0800
Subject: [PATCH] Reduce lang_not_found message

---
 cli/src/util.rs | 19 +------------------
 1 file changed, 1 insertion(+), 18 deletions(-)

diff --git a/cli/src/util.rs b/cli/src/util.rs
index fd4f4699..7d0e8987 100644
--- a/cli/src/util.rs
+++ b/cli/src/util.rs
@@ -28,25 +28,8 @@ pub fn lang_not_found_for_path(path: &Path, loader_config: &LoaderConfig) -> Str
     format!(
         indoc! {"
             No language found for path `{}`
-
-            If a language should be associated with this file extension, please ensure the path to `{}` is inside one of the following directories as specified by your 'config.json':\n\n{}\n
-            If the directory that contains the relevant grammar for `{}` is not listed above, please add the directory to the list of directories in your config file, {}
         "},
-        path,
-        path,
-        loader_config
-            .parser_directories
-            .iter()
-            .enumerate()
-            .map(|(i, d)| format!("  {}. {}", i + 1, d.display()))
-            .collect::<Vec<_>>()
-            .join("  \n"),
-        path,
-        if let Ok(Some(config_path)) = Config::find_config_file() {
-            format!("located at {}", config_path.display())
-        } else {
-            String::from("which you need to create by running `tree-sitter init-config`")
-        }
+        path
     )
 }
 
-- 
2.41.0.windows.1

